### PR TITLE
Add redirection notices to tutorials

### PR DESCRIPTION
**Pull request by ahmetb**
This adds an EXCLUDE_FROM_DOCS section (removed by docs script, to be added on https://github.com/kubernetes/kubernetes.github.io/pull/4291) to the
tutorials and fixes relative-link issues in the tutorial contents.

/cc @sebgoa

### CODE REVIEW

File: cassandra/README.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
+<!-- EXCLUDE_FROM_DOCS BEGIN -->
+
+> :warning: :warning: Follow this tutorial on the Kubernetes website:
```
Comment (sebgoa) [inline cassandra/README.md]: I would add "Otherwise some of the URLs will not work properly".

File: guestbook/README.md [hunk 2]
```diff
@@ -47,7 +53,7 @@ $ kubectl cluster-info
 
 If you see a url response, you are ready to go. If not, read the [Getting Started guides](http://kubernetes.io/docs/getting-started-guides/) for how to get started, and follow the [prerequisites](http://kubernetes.io/docs/user-guide/prereqs/) to install and configure `kubectl`. As noted above, if you have a Google Container Engine cluster set up, read [this example](https://cloud.google.com/container-engine/docs/tutorials/guestbook) instead.
 
-All the files referenced in this example can be downloaded in [current folder](./).
+All the files referenced in this example can be downloaded [from GitHub](https://git.k8s.io/examples/guestbook).
```
Comment (sebgoa) [inline guestbook/README.md]: I would add somewhere here: 

```
git clone https://github.com/kubernetes/examples
cd examples
```

File: mysql-wordpress-pd/README.md [hunk 3]
```diff
@@ -31,10 +38,10 @@ your editor added one.
 
 ```shell
 tr --delete '\n' <password.txt >.strippedpassword.txt && mv .strippedpassword.txt password.txt
-kubectl create -f https://raw.githubusercontent.com/kubernetes/kubernetes/master/examples/mysql-wordpress-pd/local-volumes.yaml
+kubectl create -f https://raw.githubusercontent.com/kubernetes/examples/master/mysql-wordpress-pd/local-volumes.yaml
```
Comment (sebgoa) [inline mysql-wordpress-pd/README.md]: it is a bit "funny", none of these examples are consistent in the way they ask you to fetch the manifests.

File: guestbook/README.md (hunk 2 above)
Comment (ahmetb) [inline guestbook/README.md]: The actual content will be later cleaned up by @cody-clark later, so I'm just doing the bare minimum to keep the tutorial working.

File: mysql-wordpress-pd/README.md (hunk 3 above)
Comment (ahmetb) [inline mysql-wordpress-pd/README.md]: I'll cc @cody-clark we should probably unify the way tutorial materials are downloaded later when we start editing these tutorials.

**Comment by ahmetb** [comment]
Added 

    > Otherwise some of the URLs will not work properly.

merging.

**Pull request merged**